components/eamxx/tests/coupled/dynamics_physics/homme_shoc_cld_spa_p3_rrtmgp_dp/homme_shoc_cld_spa_p3_rrtmgp_dp.cpp

```diff
@@ -0,0 +1,64 @@
+#include "catch2/catch.hpp"
```

Comment:
We ought to create some sort of common-ad-testing lib, so that we build this file only once. We have almost the exact identical file in all coupled/uncoupled test directories...